archive-journal.php

```diff
@@ -0,0 +1,51 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+  <div class="row">
```

Comment:
yes i noticed somewhere that one of the main divs never gets closed :[ I'll fix in this pull.